src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -43,18 +43,29 @@
 import org.gradle.internal.impldep.com.google.common.annotations.VisibleForTesting;
 
 import static org.gradle.api.plugins.JavaPlugin.COMPILE_CLASSPATH_CONFIGURATION_NAME;
+import static org.gradle.api.plugins.JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME;
 
 public class DependenciesFinder
 {
   private static final String RELEASE_COMPILE_LEGACY_CONFIGURATION_NAME = "_releaseCompile";
 
+  private static final String RELEASE_RUNTIME_APK_LEGACY_CONFIGURATION_NAME = "_releaseApk";
+
+  private static final String RELEASE_RUNTIME_LIBRARY_LEGACY_CONFIGURATION_NAME = "_releasePublish";
```

Comment:
This would be the legacy equivalent of "runtime" for Android libraries (not apps).

Reply:
> This would be the legacy equivalent of "runtime" for Android libraries (not apps).

Out of documentation around this out of curiosity? I can't seem to find much mentioning "_releaseApk" or "_releasePublish"

Reply:
> > This would be the legacy equivalent of "runtime" for Android libraries (not apps).
> 
> Is there documentation around this out of curiosity? I can't seem to find much mentioning "_releaseApk" or "_releasePublish"

I couldn’t find docs about it, and I’m not surprised as those configurations where deprecated and deleted years ago in the Android gradle plugin.

What I did to find the right configurations name was using the “dependencies” task in an Android project using Gradle 3.3 and see the dependencies trees for each configuration.